cpp/example_code/cross-service/serverless-aurora/ItemTrackerHTTPHandler.cpp

```diff
@@ -0,0 +1,343 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+/**
+ *  ItemTrackerHTTPHandler.h/.cpp
+ *
+ *  The code in these file implements the HTTP server portion of the Aurora serverless
```

Comment:
these file --> these files **or** this file

Aurora serverless --> Amazon Aurora Serverless